test/e2e/framework/pv_util.go

```diff
@@ -233,15 +290,18 @@ func CreatePVCPV(c clientset.Interface, pvConfig PersistentVolumeConfig, pvcConf
 
 	By(fmt.Sprintf("Creating a PVC followed by a%s PV", preBindMsg))
 	// instantiate the pvc
-	pvc = CreatePVC(c, ns, pvc)
+	if pvc, err = CreatePVC(c, ns, pvc); err != nil {
```

Comment:
Just because we do it 170 times in the "wrong" way, doesn't make it good. We are not particularly good in writing idiomatic go code.